lib/Elastica/Multi/Search.php

```diff
@@ -27,18 +32,20 @@ class Search
     protected $_options = array();
 
     /**
-     * @var \Elastica\Client
+     * @var array|\Elastica\Search[]
      */
-    protected $_client;
+    protected $_searches = array();
 
     /**
      * Constructs search object.
      *
      * @param \Elastica\Client $client Client object
+     * @param MultiBuilderInterface $builder
      */
-    public function __construct(Client $client)
+    public function __construct(Client $client, MultiBuilderInterface $builder = null)
     {
-        $this->setClient($client);
+        $this->_builder = $builder ?: new MultiBuilder();
```

Comment:
Yeah, its a problem. I'd prefer not to have any in the Client at all because the Client shouldn't know about this stuff. I'm contemplating removing the Client option and forcing it to be set in the constructor of both the normal and multi Search instances.